pkg/sriov/poolconfig.go

```diff
@@ -255,6 +255,29 @@ func (builder *PoolConfigBuilder) WithMaxUnavailable(maxUnavailable intstrutil.I
 	return builder
 }
 
+// WithRdmaMode method sets rdmaMode to shared/exclusive.
+func (builder *PoolConfigBuilder) WithRdmaMode(mode string) *PoolConfigBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	if mode == "" {
+		builder.errorMsg = "RdmaMode cannot be empty"
```

Comment:
could you please add a glog statement to both of the mode check if statements? something like
```go
glog.V(100).Info("PoolConfig RdmaMode cannot be empty")
```